### PR TITLE
Tico thepsourinthone

### PR DESCRIPTION
**Pull request by ticotheps**
Hey @tryingtokeepup,

Here is my submission for today's assignment - Intro to Python I.

### CODE REVIEW

File: src/03_modules.py [hunk 1]
```diff
@@ -6,26 +6,29 @@
 """
 
 import sys
+print("sys.argv = ", sys.argv)
```
Comment (tryingtokeepup) [inline src/03_modules.py]: Very clean, I really like how you always parse the commands from the user and make it really readable.

File: src/03_modules.py [hunk 2]
```diff
@@ -6,26 +6,29 @@
...
+print("Number of Total Arguments: ", len(sys.argv))
```
Comment (tryingtokeepup) [inline src/03_modules.py]: Very nice, interesting usage of length here!

File: src/03_modules.py [hunk 3]
```diff
@@ -6,26 +6,29 @@
...
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
-# YOUR CODE HERE
+for i in range(len(sys.argv)):
+    print("Command Line Argument #", i+1, ":", sys.argv[i])
```
Comment (tryingtokeepup) [inline src/03_modules.py]: HAH! clever. skip the intial argument, because you want to start with the first one :D

File: src/03_modules.py [hunk 4]
```diff
@@ -6,26 +6,29 @@
...
-# YOUR CODE HERE
+print("Current Version of Python in Use: ", sys.version)
 
 
 import os
 # See the docs for the OS module: https://docs.python.org/3.7/library/os.html
 
 # Print the current process ID
-# YOUR CODE HERE
+print("Current Process ID: ", os.getegid())
```
Comment (tryingtokeepup) [inline src/03_modules.py]: Oops. I think this is `print(os.getpid())`. Try checking this again!

File: src/08_comprehensions.py [hunk 5]
```diff
@@ -36,6 +48,6 @@
 x = input("Enter comma-separated numbers: ").split(',')
 
 # What do you need between the square brackets to make it work?
-y = []
+y = [x[i] for i in range(1,len(x)) if (i % 2 == 0)]
```
Comment (tryingtokeepup) [inline src/08_comprehensions.py]: great work! try this too sometime: `y = [i for i in x if int(i) % 2 == 0]`

File: src/14_cal.py [hunk 6]
```diff
@@ -21,4 +21,37 @@
...
+elif len(args) == 3:
+    month = int(args[1])
+    year = int(args[2])
+else: 
+    print("Error: Must be in format [month] [year]")
+    exit()
+
+calendar.TextCalendar().prmonth(year,month)
+
+
```
Comment (tryingtokeepup) [inline src/14_cal.py]: hella clean.

**Comment by tryingtokeepup** [comment]
Excellent job, merging. Tico, well done.

File: src/03_modules.py (hunk 4 above)
Comment (ticotheps) [reply src/03_modules.py]: I changed it! Thanks for the catch!

**Pull request merged**